warehouse/query-core/src/main/java/datawave/query/QueryParameters.java

```diff
@@ -43,7 +43,7 @@ public class QueryParameters {
     /**
      * ??
      */
-    public static final String BLACKLISTED_FIELDS = "blacklisted.fields";
+    public static final String DISALLOWED_FIELDS = "disallowed.fields";
```

Comment:
allow/deny?  include/exclude? allow/disallow?  just curious if we're being consistent throughout.

Reply:
I'm partial to `include`/`exclude` when updating our language. Just my two cents.

Reply:
Removing these changes as updating terms should probably be its own effort